<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="RequestFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>781</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="methodSelect">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>79</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>GET</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>POST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PUT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DELETE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HEAD</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="urlText">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="reqruleSelect">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>50</y>
       <width>79</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ALLOW</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DENY</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="reqruleButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>50</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QLabel" name="requestTitle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>请求规则添加</string>
     </property>
    </widget>
    <widget class="QLabel" name="methodLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Method:</string>
     </property>
    </widget>
    <widget class="QLabel" name="urlLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Url:</string>
     </property>
    </widget>
    <widget class="QLabel" name="reqruleLable">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rule:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="responseFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>781</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="responseTitle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>响应规则添加</string>
     </property>
    </widget>
    <widget class="QLabel" name="typeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="typeSelect">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>79</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>APPLET</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DOC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>APK</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="ipLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Url:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ipText">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="resruleLabel">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>60</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rule:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="resruleSelect">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>60</y>
       <width>79</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ALLOW</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DENY</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="resruleButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>60</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ruleFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>781</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="ruleTitle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>目前规则</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>200</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>REMOVE</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="ruleTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>620</width>
       <height>191</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="proxyTitle">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>透明代理防火墙用户界面</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
